ui/app/components/ui/popover/popover.component.js

```diff
@@ -59,12 +74,13 @@ const Popover = ({ title, subtitle, children, footer, footerClassName, onBack, o
 
 Popover.propTypes = {
   title: PropTypes.string.isRequired,
-  subtitle: PropTypes.string.isRequired,
+  subtitle: PropTypes.string,
   children: PropTypes.node,
   footer: PropTypes.node,
   footerClassName: PropTypes.string,
   onBack: PropTypes.func,
   onClose: PropTypes.func.isRequired,
+  CustomBackground: PropTypes.node,
```

Comment:
This was resulting in PropType warnings

```suggestion
  CustomBackground: PropTypes.func,
```

Reply:
addressed in 9bfe721f8